apps/src/applab/applab.js

```diff
@@ -1445,7 +1413,7 @@ Applab.onPuzzleComplete = function (submit) {
         level: level.id,
         result: levelComplete,
         testResult: Applab.testResults,
-        submitted: !!submit,
+        submitted: submit ? "true" : false,
```

Comment:
Ahh yes. Since you're in this code already, it might be a good time to fix that and have it always return one or the other.